recipes/unmicst/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "unmicst" %}
+{% set version = "2.6.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/labsyspharm/UnMicst/archive/refs/tags/{{ version }}.tar.gz
+  sha256: d4da6cf35c6a2ab14009038c4308b85d458070dfbe158dfa02af173d9503b912
+  folder: UnMicst-{{ version }}
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - mkdir -p "{{ SP_DIR }}/{{ name }}"
```

Comment:
Can you pip install your package instead? What are we copying here? This is not very readable imho and could potentially include a lot of file that are not needed.

Reply:
This package is not available on PyPI.

Reply:
@jaidevjoshi83  But can't you install the source with `pip install <source_dir>` ?